__tests__/nc_news.test.js

```diff
@@ -152,22 +152,26 @@ describe('GET /api/articles/:article_id', () => {
 			return request(app)
 				.get('/api/articles/1')
 				.then(({ body }) => {
-					const article = body.article[0];
-					expect(typeof article.article_id).toBe('number');
-					expect(typeof article.title).toBe('string');
-					expect(typeof article.topic).toBe('string');
-					expect(typeof article.author).toBe('string');
-					expect(typeof article.body).toBe('string');
-					expect(typeof article.created_at).toBe('string');
-					expect(typeof article.votes).toBe('number');
-					expect(typeof article.article_img_url).toBe('string');
+					const article = body.article;
+					const articleFormat = {
+						author: expect.any(String),
+						title: expect.any(String),
+						article_id: expect.any(Number),
+						body: expect.any(String),
+						topic: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+						comment_count: expect.any(Number),
+					};
+					expect(article).toMatchObject(articleFormat);
```

Comment:
This ticket demands a new test, rather than a refactor of this one - please put this back to normal and create a new test for this ticket :)

Reply:
I don't quite understand what the point of this is? If we are adding comment_count to the returned object for every use of the endpoint wont this break old tests that are looking for an object that has specific keys?